### ISSUE TITLE
Multiple-table transaction

### ISSUE THREAD

**Issue by marcin-workfellow** [delta-io/delta#832]
It will be nice to have the transaction working for multiple tables.
If the data entry in one of the tables fails - all the changes should be roll-back-ed.

**Comment by scottsand-db** [comment]
Hey @marcin-workfellow thanks for making this issue. This feature isn't supported right now, but if this post gets enough upvotes and engagement over time, then we can think about implementing this feature.

In the meantime, how to overcome this problem was recently asked during our Delta Community Office Hours on Nov 11, 2021, and you can check out the question & answer [here](https://youtu.be/SOgI8gx1tjE?t=1070) (17:50 mark)

**Comment by prakharjain09** [comment]
The [managed-commits](https://github.com/delta-io/delta/issues/2598) effort is changing the atomicity point from object stores to an external commit-owner. This will open up the possibility to commit to multiple tables atomically using an external system. Extending the managed-commit API to support multi-table transactions will be the next step once managed-commits is ready.

**Comment by foss-contributor** [comment]
@marcin-workfellow, thank you for your contribution! This issue has been inactive for over 18 months and will be closed in one week. To keep it open, please leave a comment on this issue or contact us in the `#contributors` channel on [Delta Lake Slack](https://go.delta.io/slack). We appreciate your involvement in Delta Lake!
